engine/gamesys/src/gamesys/resources/res_model.cpp

```diff
@@ -276,6 +276,11 @@ namespace dmGameSystem
                 }
 
                 texture_info->m_SamplerNameHash = dmHashString64(texture->m_Sampler);
+                if (!texture_info->m_SamplerNameHash) // old content
+                {
+                    // Then we'll fallback to using the order of each sampler
+                    texture_info->m_SamplerNameHash = dmRender::GetMaterialSamplerNameHash(info.m_Material->m_Material, t);
+                }
```

Comment:
Supporting old content which isn't stored in the new format.
Then we only have the order of the samplers in the material to go on (as we used to before the multimaterial update)